src/main/pages/user-management/resource-management.adoc

```diff
@@ -11,77 +11,82 @@ folder: user-management
 
-Since workspaces are the critical for users in the development flow, resources are mostly related to workspaces and runtimes.
+The amount and usage refers to the amount and usage of resources for workspaces and runtimes.
 
-Multi-user Che supports four types of resources:
+Multi-user Che supports the following type of resources:
 
-* *RAM* - defines the amount of RAM which can be used by running workspaces at the same time;
-* *Timeout* - defines the period of time that is used to control idling of user workspaces;
-* *Runtime* - defines the amount of workspaces which user can run at the same time;
-* *Workspace* - defines the amount of workspaces which user can have at the same time.
+* *RAM* -  Amount of RAM that workspaces use.
+* *Timeout* - Period of time that workspaces are idle.
+* *Runtime* - Number of workspaces that users run at the same time;
+* *Workspace* - Number of workspaces that users can have at the same time.
 
 [id="resource-api"]
 == Resource API
 
 *Total resources*
 
-`GET resource/${accountId}:` allows getting information on total resources list that is allowed to use by a specified account;
+`GET resource/${accountId}:` Gets the total resources allowed by an account;
 
 *Used resources*
 
-`GET resource/{accountId}/used:` allows getting information on used resources list that is consumed by the specified account;
+`GET resource/{accountId}/used:` Gets the resources used by an account;
 
 *Available resources*
 
-`GET resource/${accountId}/available:` allows getting information on available resources list that is not used, so if there are no used resources at the moment of the request, the result must equal total resources, otherwise, the result must contain deduction of used resources from total resources list;
+`GET resource/${accountId}/available:` Gets the resources that are available and not used by an account. If no resources are used, the available resources equal the total resources.  If resources are used, the available resources equals total resources minus used resources. 
 
 *Resource details*
 
-`GET resource/{accountId}/details:` allows getting detailed information about resource list that is provided to the specified account. The result of the request contains information about resource providers and the beginning/end of period of use.
+`GET resource/{accountId}/details:` Gets detailed information about the resources that are available for an account. The detailed information include:  resource providers, resource start time, and resource end time.
 
-The more detailed specification of the response objects and required parameters are available at Swagger page: `${che-host}/swagger/#/resource`.
+See the Swagger page: `${che-host}/swagger/#/resource`, for more information about the response objects and required parameters.
```

Comment:
Separating this information using a colon and a comma creates illogical clauses. I'd reword each instance of this sentence with something like the following:

```adoc
For more information about the response objects and required parameters, see the Swagger page at `${che-host}/swagger/#/resource`.
```